.github/workflows/dashboards-lighthouse-test.yml

```diff
@@ -0,0 +1,133 @@
+name: Dashboards lighthouse tests
+on:
+  pull_request:
+    branches:
+      - 'master'
+    paths:
+      - 'test/cypress/performance/**'
+      - 'samples/cypress/dashboards/**'
+      - 'samples/dashboards/**'
+      - 'ts/Dashboards/**'
+      - 'ts/Data/**'
+      - 'css/**'
+      - '.github/workflows/**'
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
+jobs:
+  # cypress_lighthouse_base:
```

Comment:
Why do we have commented lighthouse here?

Reply:
That's for the next PR, which should add the comparison with master. It requires the cypress test to be on master to work (without any hacks at least), so this PR should be merged before activating the `cypress_lighthouse_base ` job